tests/integration/targets/role_uki/vars/testimage.yml

```
@@ -0,0 +1,2 @@
+---
+test_image: 
```

This file should be replaced with a platform list and the URIs to get images, so we can test on multiple platforms. This also allows us to track the testing platforms instead of requiring them to be configured before each test. It might look like this:

```suggestion
- name: Fedora 41
  url: 'https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2'

- name: Debian 12
  url: 'https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-nocloud-amd64.qcow2'
```